scripts/build/companion_tools/050-make.sh

```diff
@@ -23,7 +23,7 @@ do_companion_tools_make_build() {
     CT_DoExecLog ALL ${make}
     CT_DoExecLog ALL ${make} install
     if [ "${CT_COMP_TOOLS_make_gmake}" = "y" ]; then
-        CT_DoExecLog ALL ln -sv ${make} "${CT_BUILDTOOLS_PREFIX_DIR}/bin/gmake"
+        CT_DoExecLog ALL ln -sv make "${CT_BUILDTOOLS_PREFIX_DIR}/bin/gmake"
```

Comment:
I didn't say your change wasn't tested. What I said is that scripts executed **after** this one will still invoke wrong make. For example, look at `scripts/build/cc/100-gcc.sh`:
````
CT_DoExecLog CFG ${make} ${JOBSFLAGS} configure-libiberty
````
This will again invoke /usr/bin/make rather than the one in `buildtools`.